apps/fulfillment/mock/db.json

```diff
@@ -0,0 +1,888 @@
+{
+  "authorize": [
+    {
+      "code": "def5020069391194b8b5daf70177cb423146e95371b573daea480823deb1b04da95dd04588c8e7807822cecddd4846c33b5bb07674fbfd9c832f3f7d8f9f531666719a62890143ee3e2e91e32383e885a0deb13e7434503dd2541ee97470c7856f9d767b6405ca837c7cd68a115018e0291415166436b9ceb8dd459f578e0959051cce067fd4ddd66c721f2b1d8451d0c40d3842e9c7adb8c2905ab70afa61371d0c6ff79bf40e772dcd9978ddd7b579ad160b3a95a09e7a491108d63b1280a5cd782ee3261381cd6dccf7934ca5b68b612763cbb8ce1301a7fa49e5de676ab6e0602c96eefcd56cbf1d4c0e14d5281d40996e9c8b2a37a0bb71261ef340970370534424c9550ad91413232eda601df00a460f400d827eb46c4b2bf6b9efa6b13f441a1abb04a87f9dec9011d922f9ea3a2fb0b911d8ab3c2168809f90a598f5580bf815105734cbd8452427eb292ff27a839e2b9a7a4f45efe02e66a209c54ee848c3434915bf7c3f4be4cbdeb57f60587127c3da759a3a6280902e2ca9aae30d407ba9f0a9654d18f987b2c5f264755ea169f81b9b0e33e95a947ff31369495f60c503db438a5e92c618bfad0aa23735eb30c65492add5ab50c6749d51df9e3ab299dea12988a49f750a56e47501c05bcc9e6708f5949000703c8586aedc51bd4d09406bcd2032f196e54760d681c41751c9e3c31b8a830bfc5e12b7a98c76d2d17835d22b",
+      "state": null
+    }
+  ],
+  "token": [
+    {
+      "access_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiJmcm9udGVuZCIsImp0aSI6ImM4YjE4YTRlZTZlOWNhNzczMzFlZjliZDM1MjYyOWYwZTMwODNlYjQ1OWI2YjM5NDc1NmViNGIzNDY1NzgyMGViZjAyYmRhM2I0YTVjZDk0IiwiaWF0IjoxNjc3NDkwNzM4LjY0OTM4LCJuYmYiOjE2Nzc0OTA3MzguNjQ5MzgyMSwiZXhwIjoxNjc3NTE5NTM4LjYzMDk3NSwic3ViIjoie1widXNlcl9yZWZlcmVuY2VcIjpudWxsLFwiaWRfdXNlclwiOjN9Iiwic2NvcGVzIjpbXX0.d5mfJdy9tnVF41HnuMEhEbqkv4kZOA13JEACefvIgZgEOPGGeFCNC1-o3vHaEMXzKlPtwfIt2FEUl05xWFyEMxSgMyL39Ek8eJJOwuqvgMgOqzliysOEQmRxEcmiTIKXl6fI46IznJq2dH7ebjxDfDE5heENOMIKxuWFzeos09x_JEBQaUKH-HEB-Wke9jZ4x8SsDkDdS2xaNBWTzlS7Ym3aQjP0wQxAG5AfAhQOjntNXLIFXHZEs03oWpIVmUTWmokczScy2awwcussrVQlwtH3vtEntWaP670ARBsH4SEDfkozo-eQ_oLbSzuZKYuGmn3UQZDbvS_IBQqO6-Kv_A",
+      "token_type": "Bearer",
+      "expires_in": 28800,
+      "refresh_token": "def5020029b7becd9092d801c6a68fc64e3b145dc518e0a06caa930a848a0b9efd63ebf72af6738c459e1668023b47a25998f1dddb46a395bd491c10b9d0505160f506e7f4d704c25e34bcd8887c79a0a29808546bfb1ce1f7b062e8186827e6eade6c5ee90cbeac540475cfc5ebdc3f9c1cc6e2fc91657d11ea63421ea1632f5e3fe504d12443697edb6e54fb8508a366238d637892df5b9a8b8778a2dc19de28a72476cdb7328aed976f7d08ecfcdf3bc35a1a7ecfefa3fd9560d45177ce2be9be23a70b3b53f0ddf92014b1cf16e76fab102bde3327585ce2d5d75568fed534db4a43fc9b380f3a98a63e67ae440a955bec572043f09ab21550d2ab21524bbc650d88bdb11a670b39a15361266ca3545752fa95921849bf306815d57e4e50e33b49f6a6062b8c6fdee8a563e59618a217f6672286365866b60addfdc22e626dd4f5bd306e8d91d7d2d0e122d83701773714c751f0eac0787f7b7e71f1f1ea5110523aec4645d5b3eea43de5cbf13bdf0f89c2a0dcab4d09d4470c18ef08a4944ddb11eca69b8fe1dd443b74067b8102383cb3fa6fffda2cdf8d46c6acae36ef6e673fb850583f0724e9c1fe9a04060dbd5c000b4e8160ff92cd7f256591ce17daa175b63070c31ae3cd0b7201"
+    }
+  ],
```

Comment:
Btw `/token` response should be changed to just object without any arrays as it's part of the Oauth spec.